src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/eye/eye_kernel_ref.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "kernel_base_opencl.h"
+
+namespace kernel_selector {
+
+struct eye_params : public base_params {
+    eye_params() : base_params(KernelType::EYE) {}
+
+    std::vector<std::int32_t> diagonal_index;
```

Comment:
why it is vector? isn't it always scalar?

Reply:
yes, it is always scalar, removed vector.